scheduler/dev-config.edn

```diff
@@ -1,6 +1,17 @@
 {:port 12321
  ;; We'll set the user to vagrant, since that's the default for many Vagrant-based Mesos setups
  :authorization {:one-user "vagrant"}
+
+ ;; What function should be used to perform user authorization?
+ ;; See the docstring in cook.authorization for details.
+ :authorization-fn cook.authorization/configfile-admins-auth
```

Comment:
OTOH, we don't know a priori what a user's custom authorization-fn does; it might perform any arbitrary computation. Might it need some information from the general config file to do that? Do we want to specify that it's only allowed to use data from that particular key to do authorization?